modules/troubleshooting-network-observability-controller-manager-pod-out-of-memory.adoc

```diff
@@ -6,26 +6,41 @@
 [id="controller-manager-pod-runs-out-of-memory_{context}"]
 = Network Observability controller manager pod runs out of memory
 
-You can increase memory limits for the Network Observability operator by patching the Cluster Service Version (CSV), where Network Observability controller manager pod runs out of memory.
+You can increase memory limits for the Network Observability operator by editing the `spec.config.resources.limits.memory` configuration in the `Subscription` object.
 
 .Procedure
 
-. Run the following command to patch the CSV:
+. In the web console, navigate to *Operators* -> *Installed Operators*
+. Click *Network Observability* then select *Subscription*.
+. From the *Actions* menu, choose *Edit Subscription*.
+.. Alternatively, you can use the CLI to open the YAML configuration for the `Subscription` object by running the following command:
 +
 [source,terminal]
 ----
-$ oc -n netobserv patch csv network-observability-operator.v1.0.0 --type='json' -p='[{"op": "replace", "path":"/spec/install/spec/deployments/0/spec/template/spec/containers/0/resources/limits/memory", value: "1Gi"}]'
+$ oc edit subscription netobserv-operator -n openshift-netobserv-operator
 ----
+. Edit the  `Subscription` object to add the `config.resources.limits.memory` and set the memory to account for your needs:
 +
-.Example output
+[source,yaml]
 ----
-clusterserviceversion.operators.coreos.com/network-observability-operator.v1.0.0 patched
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: netobserv-operator
+  namespace: openshift-netobserv-operator
+spec:
+  channel: stable
+  config:
+    resources:
+      limits:
+        memory: 800Mi     <1>
+      requests:
+        cpu: 100m
+        memory: 100Mi
+  installPlanApproval: Automatic
+  name: netobserv-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: network-observability-operator.v1.4.1
```

Comment:
My experience with stating versions is they are a bit of a tricky value to maintain. I would go with a user-repolaced value to avoid the task of having to update this value for the latest version. Sometimes these updates get skippes if a team member moves onto something else.

Reply:
@jotak is it OK to say <network_observability_latest_version> here instead of having v1.4.1? I can understand the maintenance worry that @dfitzmau points out.

Reply:
This part isn't meant to be edited by the user, they need to leave it as it is, but yes they could see something different depending on which version they installed at first (but it won't necessarily be the latest)
I agree that it won't age well after a couple of releases, I'm fine if you want to replace it with a generic text / annotation